packages/orbit-components/src/Modal/Modal.stories.tsx

```diff
@@ -562,3 +562,43 @@ Rtl.story = {
     info: "This is a preview of this component in RTL setup.",
   },
 };
+
+export const VisualTest = () => {
+  const { Container, onClose } = useModal();
+  const content = "Lorem ispum dolor sit amet.";
+
+  return (
+    <Container>
+      <Modal onClose={onClose} mobileHeader size={SIZES.NORMAL}>
+        <ModalHeader
```

Comment:
Agreed. Tbh I didn't go into all the possible variations of Modal because, in the end, it is impossible to verify all possibilities of a Modal in one story. But this one is definitely possible, so I've added as you suggested, thanks!